nodejs/clients.js

```diff
@@ -0,0 +1,73 @@
+const { readMasterKey, ClientBuilder } = require("./helpers")
+
+const localMasterKey = readMasterKey()
+
+const clientBuilder = new ClientBuilder({
+  kmsProviders: {
+    local: {
+      key: localMasterKey
+    }
+  }
+})
+// Wrapping everything in an async iife to allow use of async/await
+;(async () => {
```

Comment:
Nitpick: 
Should the semicolon be on line 11 instead or is there a formatting standard that prefers them in front of the next command?

Reply:
This is `prettier` formatting it like this, which I have configured globally. The reasons it does this can be traced back to https://github.com/prettier/prettier/pull/1129#issuecomment-292243414

Reply:
Seems like it belongs in the `uglier` package. `¯\_(ツ)_/¯`
This also means that if you were removing the statement above, you would also need to modify this line which makes git blame less useful. Seems like formatting rules such as trailing commas contradict this type of formatting, as they provide the benefit of not requiring a line update to the original final item when appending additional items.